service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -55,6 +56,13 @@ fi
 sudo -u "${JUPYTER_USER}" sh -c "curl -s https://get.nextflow.io | bash"
 sudo mv nextflow /usr/bin/nextflow
 
+# Install cromwell
+readonly CROMWELL_LATEST_VERSION="81"
+sudo -u "${JUPYTER_USER}" sh -c "mkdir -p /home/${JUPYTER_USER}/cromwell"
+cd /home/${JUPYTER_USER}/cromwell
+sudo -u "${JUPYTER_USER}" sh -c "curl -LO https://github.com/broadinstitute/cromwell/releases/download/${CROMWELL_LATEST_VERSION}/cromwell-${CROMWELL_LATEST_VERSION}.jar | bash"
```

Comment:
curious why this pipes to bash at the end. Do we execute something in the .jar file?

Reply:
good question. it was just copied from above. i removed it.